demos/widgets/lv_demo_widgets_analytics.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file lv_demo_widgets_analytics.h
+ *
+ */
+
+#ifndef LV_DEMO_WIDGETS_ANALYTICS_H
+#define LV_DEMO_WIDGETS_ANALYTICS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_demo_widgets.h"
+#if LV_USE_DEMO_BENCHMARK
```

Comment:
```suggestion
#if LV_USE_DEMO_WIDGETS
```
Same issue in the other files